packages/nx/src/devkit-exports.ts

```diff
@@ -41,6 +41,10 @@ export type {
  * @category Workspace
  */
 export { Workspaces } from './config/workspaces';
+/**
+ * @category Workspace
+ */
+export { getExecutorInformation } from './command-line/run/executor';
```

Comment:
Don't export this because no one uses it.

Reply:
removed